content/tutorials/trainapp-fitbit.textile

```diff
@@ -0,0 +1,550 @@
+---
+title: Building a Clockface app for Fitbit using Ably
+section: tutorials
+index: 95
+---
+
+In this tutorial, we will see how to use the "Ably Realtime client library":/realtime to build a realtime London Tube schedule clockface app for Fitbit. Ably enables realtime data sharing using Pub/Sub messaging architecture via a concept called "channels":https://www.ably.io/channels.
+
+For the purpose of this tutorial, we'll not discuss storage of messages or other server-side mechanisms.
+
+
+"Getting started with Fitbit SDK":https://dev.fitbit.com/getting-started/ is a guide to get started for developing apps and clockface apps for Fitbit.
+
+<%= partial 'tutorials/_step-1-setup-free-account' %>
+
+h2(#create-clockface-app). Step 2 - Create a clock face 
+
+Prerequisites:
+
+* Make sure "Node and NPM are installed":https://nodejs.org/en/download/ on your system.
+* You will not need to install the Fitbit SDK separately as we'll be using NPX (that comes bundled with NPM) for doing this.
```

Comment:
Is this meant to be a sub-point of the above point? If so an extra indent would be good.